### PR TITLE
Issue 71: Stats block as slide fragment

### LINKED ISSUE #71: Stats block as slide fragment
image

The content of the stats change when user change the slide. So, we should make the stats block as a content fragment.

### PR DESCRIPTION
**Pull request by allentsliu**
## Issue

Fix #71 

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
- After: https://issue-71--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1

## 📝 Description:

Screen Shot 2023-06-22 at 10 31 38 PM

### CODE REVIEW

**Comment by tmathern** [comment]
Some things:
Screenshot 2023-06-23 at 8 40 05 AM

- For the triangle badge, before stats were in I placed it above the video. This logic nows needs an update to place the badge above the stats.
- Alignment for the stats is off, text should be in the middle-height of the stats fragment.

File: blocks/slides/slides.css [hunk 1]
```diff
@@ -315,6 +317,61 @@
...
+.slides .stats-group .default-content-wrapper .border-0,
+.slides .stats-group .default-content-wrapper .border-1 {
+  margin-top: -1vw;
+  margin-bottom: -3.2vw;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 3.5vw;
+}
+
+.slides .stats-group .default-content-wrapper .border-0 {
```
Comment (tmathern) [inline blocks/slides/slides.css:371]: That one may actually need a background color now

**Review by tmathern — CHANGES_REQUESTED** [review]
See comments

**Comment by tmathern** [comment]
Something also is off with the links - there is an error logged and it seems to try to read some text as link?

**Comment by allentsliu** [comment]
@tmathern thank you for the feedback. I have addressed them. When you get a chance, could you please approve it. Thanks!

**Review by tmathern — APPROVED** [review]

**Pull request merged**